### ISSUE TITLE
Implement lchown

### ISSUE THREAD

**Issue by humphd** [filerjs/filer#620]
https://nodejs.org/api/fs.html#fs_fs_lchown_path_uid_gid_callback

**Comment by andrewkoung** [comment]
Could I try learning this and implementing it?

**Comment by humphd** [comment]
@andrewkoung sure, let's discuss in class tomorrow.

**Comment by andrewkoung** [comment]
@humphd I had forgotten to ask this in class about context.getObject(). Let's say this line of code `context.getObject(superNode.rnode, check_root_directory_node);` fires, does it look through the key-value table using `superNode.rnode` as a key and passes the value into the method `check_root_directory_node`?

**Comment by humphd** [comment]
@andrewkoung yes, that is exactly what it does.  It basically means: "get the file node for the super-node, and once we have that file node object, pass it to the check_root_directory_node function callback".
